introduction.md

```diff
@@ -4,4 +4,9 @@ Hello, I do some development sometimes.
 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?